daffodil-schematron/src/test/resources/xsd/embedded-1.dfdl.xsd

```diff
@@ -24,25 +24,26 @@
             <dfdl:format ref="GeneralFormat"/>
         </xs:appinfo>
         <xs:appinfo xmlns:sch="http://purl.oclc.org/dsdl/schematron">
-            <sch:pattern name="test-uuid">
+            <sch:pattern name="validate-uuid">
                 <sch:rule context="uuid">
-                    <sch:assert test="string-length(.)=36">uuid must be 36 chars</sch:assert>
-                    <sch:assert test="substring(.,9,1)='-' and substring(.,14,1)='-' and substring(.,19,1)='-' and substring(.,24,1)='-'">malformed uuid</sch:assert>
-                </sch:rule>
-            </sch:pattern>
-            <sch:pattern name="test-short-uuid">
-                <sch:rule context="short-uuid">
-                    <sch:assert test="string-length(.)=8">short uuid must be 8 chars</sch:assert>
+                    <sch:assert test="string-length(time_low)=8"><sch:name/> wrong length</sch:assert>
```

Comment:
So you don't need package names on these functions? Why? 

I would expect to see "fn:string-length(.) eq 8"  

Is schematron's expression language actually not XPath based?

Reply:
Ive saw it done both ways.  Will try to pin down why with a reference.

Reply:
> So you don't need package names on these functions? Why?

The default namespace is searched if the function is unprefixed, at least that is what I have gleaned from a couple random sources.

Reply:
from [iso_schematron_skeleton_for_saxon.xsl#L66-L70](https://github.com/Schematron/schematron/blob/master/trunk/schematron/code/iso_schematron_skeleton_for_saxon.xsl#L66-L70
)
```
   Another tip is that Schematron provides an
   element <iso:ns> for declaring the namespaces and prefixes used in Xpaths in 
   attribute values; it does not extend the XML Namespaces mechanism: if a name
   in an XPath has a prefix, there must be an <iso:ns> element for that prefix; if
   a name in an XPath does not have a prefix, it is always in no namespace.
```